js/main.js

```diff
@@ -1,8 +1,13 @@
 //Happy Birthday Leader! 🎇💙
+//import Papa from "papaparse"
```

Comment:
does this line do anything if it's commented? delete?

Reply:
It doesn't do anything. On second thought it confuses more than it helps, so already deleted it. o7